test/integration/run-command/run_cursor_command.test.ts

```diff
@@ -0,0 +1,53 @@
+import { expect } from 'chai';
+
+import { CommandStartedEvent, Db, MongoClient } from '../../mongodb';
+
+describe('class RunCommandCursor', () => {
```

Comment:
This is supposed to be an integration test, so you should be describing an integration-level API. The file name is `run_cursor_command`, and what you are calling in the test case is `runCursorCommand`, so I'm guessing the title here should be something along the lines of `Db.runCursorCommand()` or `runCursorCommand API`

Reply:
fixed, went with "runCursorCommand API"